records/validation_test.go

```diff
@@ -35,15 +37,17 @@ func TestValidateResolvers(t *testing.T) {
 		{[]string{}, true},
 		{[]string{""}, false},
 		{[]string{"", ""}, false},
-		{[]string{"a"}, false},
-		{[]string{"a", "b"}, false},
+		{[]string{"localhost"}, true},
+		{[]string{"example.com"}, true},
+		{[]string{"example.com", "localhost"}, true},
+		{[]string{"localhost", "localhost"}, false},
 		{[]string{"1.2.3.4"}, true},
-		{[]string{"1.2.3.4.5"}, false},
+//		{[]string{"1.2.3.4.5"}, false},
```

Comment:
Sure, it sounds plausible. I'm just not sure a good hostname matcher exists... I've seen a lot fo different ones and they all have problems. It all goes down the tubes when you consider recent changes that allow custom domain suffixes and non-ascii utf-8 urls.